### PR TITLE
D/w 7388308

### PR DESCRIPTION
**Pull request by jfeingold35**

### CODE REVIEW

File: src/commands/scanner/rule/add.ts [hunk 1]
```diff
@@ -59,4 +61,9 @@ export default class Add extends SfdxCommand {
     }
   }
 
+  private resolvePaths(): string[] {
+    // path.resolve() turns relative paths into absolute paths. It accepts multiple strings, but this is a trap because
+    // they'll be concatenated together. So we use .map() to call it on each path separately.
+    return this.flags.path.map(p => path.resolve(untildify(p)));
```
Comment (rmohan20) [inline src/commands/scanner/rule/add.ts]: Do we know if `untildify` is Windows aware? I think the concept of `~` as home directory doesn't exist in Windows. As long as this call doesn't mess up a Windows file path, we should be fine I suppose.

Comment (jfeingold35) [reply src/commands/scanner/rule/add.ts]: I did some playing with the library in Windows, and I'm satisfied that it's safe.

File: test/commands/scanner/rule/add.test.ts [hunk 2]
```diff
@@ -9,43 +9,147 @@ import path = require('path');
...
   describe('E2E', () => {
-    describe('Happy Path', () => {
-      const CATALOG_OVERRIDE = 'AddTestPmdCatalog.json';
-      const CUSTOM_PATH_OVERRIDE = 'AddTestCustomPaths.json';
+    const myLanguage = 'apex';
+    describe('Test Case: Adding individual JARs', () => {
+      // Create four temporary directories, each having a single JAR.
+      const tmpDir1 = fs.mkdtempSync(path.join(os.tmpdir(), 'foo-'));
+      const tmpJar1 = path.join(tmpDir1, 'bar1.jar');
+      fs.writeFileSync(tmpJar1, 'In the Age of Ancients, the world was unformed and shrouded by fog.');
```
Comment (rmohan20) [inline test/commands/scanner/rule/add.test.ts]: Your text is too interesting and I _had_ to find the source!
https://en.wikipedia.org/wiki/Dark_Souls

**Comment by rmohan20** [comment]
Looks good 👍

**Pull request merged**